imports/node-app/core/util/buildContext.test.js

```diff
@@ -1,49 +1,158 @@
-import mockContext from "/imports/test-utils/helpers/mockContext";
+import mockContext, { resetContext } from "/imports/test-utils/helpers/mockContext";
 import buildContext from "./buildContext";
 
-const fakeUser = {
```

Comment:
the diff isn't great here, I added a `describe` (made running tests in isolation easier) causing everything to be indented. new tests are at the bottom.